src/graphic/minimap_renderer.cc

```diff
@@ -162,25 +160,40 @@ void draw_view_window(const Widelands::Map& map,
 }
 
 // Does the actual work of drawing the minimap.
-void do_draw_minimap(Texture* texture,
+void do_draw_minimap(Texture& texture,
                      const Widelands::EditorGameBase& egbase,
                      const Widelands::Player* player,
                      const Vector2i& top_left,
-                     MiniMapLayer layers) {
+                     MiniMapLayer layers,
+                     const bool draw_full = true,
+                     uint16_t* rows_drawn = nullptr) {
```

Comment:
So is it good or bad? I'm getting mixed signals here :-P

Reply:
Pick whichever you prefer. ;-)
It would be a small, unneeded optimization(?) to remove `draw_full`. But then the caller has to read the documentation to figure out what `nullptr` means. So ... do what you want, both choices are fine. (Personally, I would remove `draw_full`. But its your code and its up to you.)